lib/octokit/client/projects.rb

```diff
@@ -0,0 +1,314 @@
+module Octokit
+  class Client
+
+    # Methods for Projects API
+    #
+    # @see https://developer.github.com/v3/repos/projects
+    module Projects
+
+      # List projects for a repository
+      #
+      # Requires authenticated client
+      #
+      # @param repo [Integer, String, Repository, Hash] A GitHub repository
+      # @return [Array<Sawyer::Resource>] Repository projects
+      # @see https://developer.github.com/v3/projects/#list-repository-projects 
+      # @example
+      #   @client.projects('octokit/octokit.rb')
+      def projects(repo, options = {})
```

Comment:
The repository endpoints can also take an id instead of login/name so I think it could get a bit messy quick.